scripts/data_contamination/README.md

```diff
@@ -65,3 +65,14 @@ with beam.Pipeline() as pipeline:
 ## Notes
 
 The beam script does not support outputting contaminated ngrams yet.
+
+
+## Docker
+
+To create and run docker image:
+
+docker build  . -t contamination_script
+docker run -e INPUT_PATH=<input_path> -e SCENARIO_DATA=<scenario_data> -e OUTPUT_STATS=<output_stats> -e INPUT_FORMAT=<input_format> --rm -it  contamination_script:latest 
```

Comment:
Indent 4 spaces so that markdown renders as a codeblock

Reply:
Done